src/core_plugins/timelion/public/register_feature.js

```diff
@@ -0,0 +1,13 @@
+import { FeatureCatalogueRegistryProvider, FeatureCatalogueCategory } from 'ui/registry/feature_catalogue';
```

Comment:
hmm, I have this code pulled down locally, but no timelion app is showing up

<img width="719" alt="screen shot 2017-11-01 at 9 15 21 am" src="https://user-images.githubusercontent.com/16563603/32277506-0fa16030-bee9-11e7-8cd6-f7d50296462c.png">

Reply:
Do you have the latest? There was a commit or two that had problems with registering timelion but those have been resolved. Let me know if a fresh pull and kibana restart do not fix these.

Reply:
ah, yes it was the latter step I was missing. Forgot to restart the sever.  All good!